src/Util/index.ts

```diff
@@ -1,5 +1,18 @@
-import { camelizeKeys } from 'humps';
+export function addDayToDate(date: string): string {
```

Comment:
what about more general function `addDaysToDate(date: string, daysCount: number)` daysCount could be also negative so it would remove days